lib/foreman_maintain/repos_manager/el.rb

```diff
@@ -0,0 +1,35 @@
+module ForemanMaintain::ReposManager
+  class El
+    include ForemanMaintain::Concerns::OsFacts
+    include ForemanMaintain::Concerns::SystemHelpers
+
+    def disable_repos(repo_ids)
+      if el8?
+        execute!("dnf config-manager --set-disabled #{repo_ids.join(',')}")
+      elsif el7?
+        execute!("yum-config-manager --disable #{repo_ids.join(',')}")
+      end
```

Comment:
given EL9 will also have dnf, should this read more like

```suggestion
      if el7?
        execute!("yum-config-manager --disable #{repo_ids.join(',')}")
      else
        execute!("dnf config-manager --set-disabled #{repo_ids.join(',')}")
      end
```